tools/chapel-py/setup.py

```diff
@@ -68,6 +81,12 @@
     "-lChplFrontendShared",
 ]
 
+if chpl_install_lib_path is not None:
+    LDFLAGS += [
```

Comment:
I don't think this check will work now. I believe in the case where there is no `chpl_install_lib_path`, it will be the literal string "None".